pkgs/development/tools/godot/default.nix

```diff
@@ -4,13 +4,13 @@
 
 stdenv.mkDerivation rec {
   name    = "godot-${version}";
-  version = "2.1.3-stable";
+  version = "2.1.4-stable";
```

Comment:
```
version = "2.1.4";
...
rev = "${version}-stable";
```

Reply:
fixed in 8b0f2b22a52ec704def97a894c6b2c408d763be5